.npmignore

```diff
@@ -15,5 +15,5 @@ examples/
 karma.conf.js
 node_modules/
 tests/
-tools/copydeps.sh
+tools/copydeps.js
```

Comment:
Is it intentional that this is now the only file in `tools` that is in the npmignore (ignoring the `.jshintrc`)?